### PR TITLE
Add command to skip without downvoting

### PR DESCRIPTION
**Pull request by rjanja**
Also "what's next", cleans up some help, minor refactoring

### CODE REVIEW

File: lib/hedwig_mopidy/responders/mopidy.ex [hunk 1]
```diff
@@ -199,14 +208,40 @@ defmodule HedwigMopidy.Responders.Mopidy do
...
+           {:ok, :success} <- Playback.next do
+            HedwigMopidy.notice_message("Skipping to next track: #{HedwigMopidy.playing_string(next_track, current_playlist)}")
+      else
+        {:error, error_message} -> error_message
+        _ -> HedwigMopidy.notice_message("No more songs are queued")
+      end
+    send message, response
+  end
+
+  hear ~r/^dj\s(what|who).*\snext$/i, message do
```
Comment (holetse) [inline lib/hedwig_mopidy/responders/mopidy.ex]: This call is not in the help, and its `regex` is too generic.

File: lib/hedwig_mopidy/responders/mopidy.ex [hunk 2]
```diff
@@ -69,12 +69,14 @@ defmodule HedwigMopidy.Responders.Mopidy do
...
 `dj playlists` lists all the available playlists
-`dj start <uri>` shuffles the playlist specified by <uri> (defaults to the Lab Zero playlist)
+`dj start` shuffles the last-played playlist, or Lab Zero default playlist
+`dj start <uri>` shuffles a publicly-available playlist
 `dj pause|stop` ceases playback
 `dj play|resume` starts playback
-`dj (what|who) ('s| is) playing` displays the currently playing track and playlist (e.g. what's playing or who is playing)
-`dj +1|:thumbsup:|:thumbsup_all:|:metal:|:shaka:|up|yes` upvotes if you like the currently playing track on the currently playing playlist
-`dj -1|:thumbsdown:|:thumbsdown_all:|down|no|skip|next` votes against the currently playing track on the currently playing playlist and skips to the next track
+`dj who's playing` or what's playing, shows track and playlist
```
Comment (holetse) [inline lib/hedwig_mopidy/responders/mopidy.ex]: The old documentation is correct, this looses other supported requests.

File: lib/hedwig_mopidy/responders/mopidy.ex [hunk 3]
```diff
@@ -199,14 +208,40 @@ defmodule HedwigMopidy.Responders.Mopidy do
...
+
+  hear ~r/^dj\svol(ume)?$/i, message do
     response = with {:ok, level} <- Mixer.get_volume do
       "The volume level is set to #{round(level/10)}"
     end
     send message, response
   end
 
-  hear ~r/^dj\svolume\s(?<level>.*)$/i, message do
+  hear ~r/^dj\svol(ume)?\s(?<level>.*)$/i, message do
```
Comment (holetse) [inline lib/hedwig_mopidy/responders/mopidy.ex:245]: Should be included in the help.

**Review by holetse — CHANGES_REQUESTED** [review]

**Comment by rjanja** [comment]
Closes #3

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 2 above)
Comment (rjanja) [reply lib/hedwig_mopidy/responders/mopidy.ex]: Emojis are not rendered within backticks, so we should either get rid of the code-formatted help or avoid specifying emoji. Here I've chosen the latter.

I would argue that a global help that shows all possible forms of every command is not that helpful and just becomes difficult to read.

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 3 above)
Comment (rjanja) [reply lib/hedwig_mopidy/responders/mopidy.ex:245]: Do we really need to tip the reader that they can either spell out volume or not? It should do the same thing

File: lib/hedwig_mopidy/responders/mopidy.ex (hunk 1 above)
Comment (rjanja) [reply lib/hedwig_mopidy/responders/mopidy.ex]: I've added help for this command. As far as the regex being too generic, I've reused what's already there for what/who's playing. We can create a new issue for being more specific in both commands if desired

Comment (holetse) [reply lib/hedwig_mopidy/responders/mopidy.ex]: I don't see why we'd want to repeat the sins of our ancestors on new code.

**Review by holetse — APPROVED** [review]

**Pull request merged**